contracts/test/v0.8/dev/FunctionsBillingRegistry.test.ts

```diff
@@ -5,9 +5,9 @@ import { Roles, getUsers } from '../../test-helpers/setup'
 import { randomAddressString } from 'hardhat/internal/hardhat-network/provider/fork/random'
 import { stringToBytes } from '../../test-helpers/helpers'
 
-let ocr2drOracleFactory: ContractFactory
+let FunctionsOracleFactory: ContractFactory
```

Comment:
is the inconsistent capitalization intentional?

Reply:
No